### ISSUE TITLE
How can I set hyperparameters for trainning?

### ISSUE THREAD

**Issue by panda7777777** [xuhongzuo/DeepOD#57]
Thank you for your excellent work!

However, I've noticed that the models may not converge on some datasets when using the `fit()` function.

Could you explain how to set the hyperparameters, such as learning rate, epochs, etc.?
